src/www/components/landingpage/LandingPage.scss

```diff
@@ -0,0 +1,612 @@
+.container {
+  background-image: linear-gradient(  rgb(129, 129, 129) ,white);
+  height: 325vh; //parallax(110vh) + 2 pages(200vh) + 15vh more empty space
+  max-width: 100vw;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Times New Roman', Times, serif;
+
+  .thirdpage{
+    
+    .phrase{
+      margin-top: 20vh;
+      margin-bottom: 10vh;
+      float: left;
+      margin-left: 1  5vw;
+      font-size: calc(3.5vh + 1vw);
+    }
+
+    .parallaxbg{
+      
+      width: 74vw;
+      height: 65vh;
+      margin-left: 13vw;
```

Comment:
Why do we only need a left margin? (even though I couldn't find how this was being used anyway since the element's width is based on the view-width)